ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -13,6 +13,7 @@ import shared.controls.chat 1.0
 Column {
     id: root
     width: parent.width
+    height: visible ? childrenRect.height: 0
```

Comment:
```
Qt Documentation

If an item within a Column is not visible, or if it has a width or height of 0, the item will not be laid out and it will not be visible within the column.
```

Reply:
@borismelnik not true. If you try printing `onVisibleChanged` you'll see:
```
qml: -----------------
qml: -----------------
qml: false 272
qml: -----------------
qml: -----------------
qml: -----------------
qml: -----------------
qml: false 272
qml: -----------------
qml: -----------------
qml: -----------------
qml: -----------------
qml: false 307
qml: -----------------
qml: -----------------
```
Maybe you're confused with ColumnLayout?

Reply:
@alexandraB99 where are you put a print?

Reply:
@alexandraB99 yeah seems like i'm stuck in some old version of Qt :(